### PR TITLE
DCOS-14415: Add Open Service action

### PR DESCRIPTION
**Pull request by jfurrow**
This PR adds an "Open Service" action to the service actions dropdown menus if the service has a web URL.

Services table:

Service detail:

**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [x] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

File: plugins/services/src/js/containers/services/ServicesTable.js [hunk 1]
```diff
@@ -177,6 +188,13 @@ class ServicesTable extends React.Component {
       },
       {
         className: classNames({
+          hidden: !this.hasWebUI(service)
+        }),
+        id: ServiceActionItem.OPEN,
+        html: 'Open Service'
```
Comment (rcorral) [inline plugins/services/src/js/containers/services/ServicesTable.js]: Can you translate this string, now that the i18n PR has merged?

File: plugins/services/src/js/containers/service-detail/ServiceDetail.js [hunk 2]
```diff
@@ -136,6 +136,17 @@ class ServiceDetail extends mixin(TabsMixin) {
 
     const actions = [];
 
+    if (service instanceof Service
+      && service.getWebURL() != null
+      && service.getWebURL() !== '') {
+      actions.push({
+        label: 'Open Service',
```
Comment (rcorral) [inline plugins/services/src/js/containers/service-detail/ServiceDetail.js]: Can you translate this string, now that the i18n PR has merged?

Comment (jfurrow) [inline plugins/services/src/js/containers/service-detail/ServiceDetail.js]: Done!

File: plugins/services/src/js/containers/services/ServicesTable.js (hunk 1 above)
Comment (jfurrow) [inline plugins/services/src/js/containers/services/ServicesTable.js]: Done!

**Comment by jfurrow** [comment]
run integration tests

**Comment by jfurrow** [comment]
run integration tests

**Comment by mlunoe** [comment]
Seems the tests are now failing:

**Comment by jfurrow** [comment]
@mlunoe Thanks for noticing that, tests are now passing!

**Comment by jfurrow** [comment]
ping @rcorral all comments addressed :)

**Pull request merged**